### ISSUE TITLE
rpiplay performance

### ISSUE THREAD

**Issue by zhanghaobb** [rahul-thakoor/balena-rpiplay#15]
This is a great tool, easy to use and quick.
I have tried under 3B+ and 4, both of them had frame lost, I can watch the video but not quite smooth, how can I improve that?

**Comment by rahul-thakoor** [comment]
Hi, how are you deploying the app? Are you using the open fleet from balenaHub or are you maintaining your own fleet?

The open fleet uses the `-l` low-latency mode already as descirbed [here](https://github.com/FD-/RPiPlay#usage). If you have your own fleet, you try to turn it off to see if it improves performance for you. You need to set the environment variable `AIRPIPLAY_LATENCY_MODE` to `0`. See more [docs here](https://www.balena.io/docs/learn/manage/variables/)

**Comment by zhanghaobb** [comment]
Thank you for the quick response.
I simply used this link: https://hub.balena.io/gh_rahul_thakoor/balena-rpiplay download the image, and flash it.
I believe it is BalenaHub.  Sorry I am still a beginner, no idea what "fleet" it is.
Let me explore more in the docs or any other recommendation to me?

**Comment by rahul-thakoor** [comment]
No issues with those warnings

**Comment by zhanghaobb** [comment]
I tested this for a few days, sometimes it works well, but after a while, video streaming became chopping with broken sound, I used pi-3B+ and pi-4, both are not ideal.  I checked the CPU performance, mostly under 50% or at lower %, not sure if it is wifi performance, can we set it to 5.8G only?

**Comment by rahul-thakoor** [comment]
Wired connection is recommended for best performance

If you want to force to 5.8G only, you will have to modify the network connection file. Please see https://www.balena.io/docs/reference/OS/network/2.x/#wifi-setup for more info

You might find this helpful too https://forums.balena.io/t/force-networkmanager-to-connect-to-2-4ghz/282156/3

**Comment by zhanghaobb** [comment]
I did use wired and Wifi.  The same.

**Comment by rahul-thakoor** [comment]
> can we set it to 5.8G only?

are you using the same SSID for both the 2.4 and 5 GHZ networks?

**Comment by zhanghaobb** [comment]
Yes, using same SSID, this is convenient for compatibility.  Let me try to disable 2.4G and see how it works.

**Comment by zhanghaobb** [comment]
There is one more issue identified, when I was trying to airplay the 2160p video, it failed to cast, it is fine and understandable because of the speed, but after a few stumbles, the pi's audio started to output constant "white noise", it can reproduce every time.
